backend/api/views.py

```diff
@@ -113,7 +113,7 @@ def experiment_populations(request, exp_slug):
 
     data = {'population': {}}
     qs = (Population.objects.filter(experiment=exp_slug)
-                            .order_by('stamp'))
+                            .order_by('stamp', 'branch'))
```

Comment:
No test changes needed here but that's only b/c we were lucky that the data was returned in the order it was entered and we happened to be assuming `control` first. So this is actually better.